src/data/set/basic.lean

```diff
@@ -1485,12 +1490,26 @@ by rw range_comp; apply image_subset_range
 lemma range_nonempty_iff_nonempty : (range f).nonempty ↔ nonempty ι :=
 ⟨λ ⟨y, x, hxy⟩, ⟨x⟩, λ ⟨x⟩, ⟨f x, mem_range_self x⟩⟩
 
+lemma image_nonempty_iff_nonempty {f:β → α} {T:set β}:
```

Comment:
This is `set.nonempty_image_iff`

https://leanprover-community.github.io/mathlib_docs/data/set/basic.html#set.nonempty_image_iff

Reply:
Agreed. Removed. Will refactor my library accordingly.